recipes/mar_greenland/meta.yaml

```
@@ -0,0 +1,19 @@
+title: 'MAR Greenland'
+description: 'Analysis-ready Zarr datasets derived from MAR netcdfs'
```

Would this be more descriptive?

```suggestion
description: 'Reconstructions of the 1900–2015 Greenland ice sheet surface mass
balance using the regional climate MAR model'
```